README.md

```diff
@@ -200,6 +200,25 @@ Read more about this Go project's [code standards](.github/CODE_STANDARDS.md).
 <br/>
 
 ## Usage
+
+
+```
+// http example
+func main() {
+
+	// Generate keys
+	keys, _ := xpriv.Generate()
+
+	// Create a client
+	_, _ = buxclient.New(
```

Comment:
Not really the best example, as you don't even store the new client in a variable, so it can't be used 🙂 

If we really want to show here some example, then IMHO it should be working example that is doing something just by copying it and running.

Reply:
client, _ := buxclient.New(
		buxclient.WithXPriv(keys.XPriv()),
		buxclient.WithHTTP("localhost:3001"),
		buxclient.WithSignRequest(true),
	)
	fmt.Println(client.IsSignRequest())
	
Can add this